src/utils/blueprints/process-template.js

```diff
@@ -1,12 +1,12 @@
 /* https://github.com/ember-cli/ember-cli/blob/v4.9.2/lib/utilities/process-template.js */
 import template from 'lodash.template';
 
-export function processTemplate(content, context) {
-  const options = {
+export function processTemplate(file, data) {
+  const settings = {
```

Comment:
I renamed variables so that the words `context` and `options` don't carry more than one meaning.